Jenkinsfile

```diff
@@ -3,5 +3,8 @@
 library("govuk")
 
 node {
-  govuk.buildProject(sassLint: false)
+  govuk.buildProject(
+    beforeTest: { sh("yarn install") },
```

Comment:
I've seen this in a few PRs now - is this something we could move into `govuk-jenkinslib`? If it detects that there is a `yarn.lock` file, it runs `yarn install`?

Reply:
Thanks for reviewing Tom 🌟 

Yes, that's a good point and something I plan to do. I thought I'd add it in at the same point that I remove sassLint from the files since I'm going to have to go through them all and edit them for that. Might do it sooner if I find a few mins.

Reply:
That makes sense, if it's on the plan to do, then I think it's fine to merge these in like this!